src/pages/singleVideo.js

```diff
@@ -1,46 +1,74 @@
 import { useParams } from "react-router-dom"
 import { Sidebar, Navbar, VideoCard } from "../components/allComp";
-import { useVideo } from "../contexts/videolist-context";
 import { AiFillLike } from "react-icons/ai";
 import { MdWatchLater, MdPlaylistAdd } from "react-icons/md";
+import { useLike, useWatchLater, useVideo } from "../contexts/allContext";
 
 export const SingleVideoPage = () => {
+    const { LikeVideoHandler, videoState: { liked }, RemoveLikeVideoHandler } = useLike()
+    const { videoState: { watchLater }, AddWatchLaterHandler, RemoveWatchLaterHandler } = useWatchLater()
     const { videoId } = useParams();
     const { video } = useVideo();
+
     function getVideoDetails(video, videoId) {
         return video.find((mp4) => mp4._id === videoId);
     }
     const mp4 = getVideoDetails(video, videoId);
-    const { title, views, creator, description, genre } = mp4;
+    const { title, views, creator, description, genre, _id } = mp4;
+    
     return (
         <div>
             <Navbar />
             <Sidebar />
-            <h1 className="video-title">{title}</h1>
-            <div className="iframe-container">
-                <iframe width="560" height="315" src={`https://www.youtube.com/embed/${videoId}`} title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-            </div>
-            <div className="video-actions">
-                <div>
-                    <p>Views: {views}</p>
+            <div className="singleVideo-grid">
+                <div className="singleVideo-div">
+                    <h1 className="video-title">{title}</h1>
+                    <iframe width="1460" height="470" src={`https://www.youtube.com/embed/${videoId}`} title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                    <div className="video-actions">
+                        <div>
+                            <p>Views: {views}</p>
+                        </div>
+                        <div className="video-btns">
+                            {liked.some((l) => l._id === _id) ?
```

Comment:
Suggestion: It is better to use better variable names for anonymous functions in array methods

Reply:
Thanks, will change the variable names